CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* Upgrade `targetSdkVersion` and `compileSdkVersion` to API 31
+* Bump `browser-switch` version to `2.X.X`
```

Comment:
Looks like it is still `api 'com.braintreepayments:browser-switch:1.1.3'` in this PR?

Reply:
Should we hold on this CHANGELOG entry for now?

Reply:
Yes this is pending the browser switch release with Android 12 support - will update the dependency and CHANGELOG after that release